tests/framework/microvm.py

```diff
@@ -280,6 +280,24 @@ def setup(self):
         os.makedirs(self._kernel_path, exist_ok=True)
```

Comment:
A general comment. This PR is ok, but you should add in a next one info about the CPU usage. You need to look from host on the CPU taken by Firecracker emulation.